src/common/entities/abstract.entity.ts

```diff
@@ -0,0 +1,31 @@
+import { Exclude } from 'class-transformer';
+import {
+  BaseEntity,
+  Column,
+  CreateDateColumn,
+  DeleteDateColumn,
+  Generated,
+  PrimaryGeneratedColumn,
+  UpdateDateColumn,
+} from 'typeorm';
+
+export abstract class AbstractEntity extends BaseEntity {
+  @PrimaryGeneratedColumn()
+  @Exclude()
+  public id: number;
```

Comment:
tip for the future: You can use uuid to the id column

Reply:
Thanks Seba!!